diff-hl-show-hunk-popup.el

```diff
@@ -1,10 +1,50 @@
 ;;; diff-hl-show-hunk-popup.el --- popup backend for diff-hl-show-hunk -*- lexical-binding: t -*-
 
+;; Copyright (C) 2020  Free Software Foundation, Inc.
+
+;; Author:   Dmitry Gutov <dgutov@yandex.ru>
+;;           Álvaro González <alvarogonzalezsotillo@gmail.com>
```

Comment:
I think it would be more correct to put just you as the author of this and other files here.